File: Prog 3 - Trabalho 2 (C++)/Vereador.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/* 
 * File:   Vereador.cpp
 * Author: alpio
 * 
 * Created on 26 de Novembro de 2016, 16:58
 */

#include<string>
#include "Partido.h"
#include "Vereador.h"

using namespace std;

void Vereador::setId (int numId){
    id = numId;
}

int Vereador::getId (){
    return id;
}

void Vereador::setColigacao (Coligacao* novaColigacao){
    coligacao = novaColigacao;
}

Coligacao* Vereador::getColigacao (){
    return coligacao;
}

void Vereador::setNome (std::string nom){
    nome = nom;
}

string Vereador::getNome (){
    return Vereador::nome;
}

void Vereador::setPartido(Partido* novoPartido){
    partido = novoPartido;
}

Partido* Vereador::getPartido (){
    return partido;
}

void Vereador::setVotos(int voto){
    votos = voto;
}

int Vereador::getVotos (){
    return votos;
}

void Vereador::setNum(std::string numero) {
    num = numero;
}

string Vereador::getNum (){
    return num;
}

void Vereador::setEleitos(bool eleit) {
    eleito = eleit;
}

bool Vereador::isEleito(){
    return eleito;
}

Vereador::Vereador() {
}

Vereador::Vereador(const Vereador& orig) {
}

Vereador::~Vereador() {
}

File: Prog 3 - Trabalho 1 (Java)/src/trabalho1/Vereador.java
package trabalho1;

public class Vereador {

		private Partido partido;
		private Coligacao coligacao;
		private String nome, num;
		private int id, votos;
		private boolean eleito;		
		
		public String getNome() {
			return nome;
		}
		public void setNome(String nome) {
			this.nome = nome;
		}

		public int getVotos() {
			return votos;
		}
		public void setVotos(int votos) {
			this.votos = votos;
		}
		public String getNum() {
			return num;
		}
		public void setNum(String num) {
			this.num = num;
		}
		public int getId() {
			return id;
		}
		public void setId(int id) {
			this.id = id;
		}
		public boolean isEleito() {
			return eleito;
		}
		public void setEleito(boolean eleito) {
			this.eleito = eleito;
		}
		public Partido getPartido() {
			return partido;
		}
		public void setPartido(Partido partido) {
			this.partido = partido;
		}
		public Coligacao getColigacao() {
			return coligacao;
		}
		public void setColigacao(Coligacao coligacao) {
			this.coligacao = coligacao;
		}

}